aws/network/bastion/main.tf

```diff
@@ -22,13 +22,29 @@ variable "owner" {
   default = ""
 }
 
+variable "open_sesame" {
```

Comment:
I would give this variable a more descriptive name, e.g. "open_ssh_to_world" and update the comment, since it could be useful independent of open-sesame (e.g. if people want to add their own permanent specific rules).